src/__tests__/keyvalue.integration.ts

```diff
@@ -0,0 +1,57 @@
+import { KeyPair, KeyValueService, Request } from "..";
```

Comment:
Is this still WIP? Should we remove this from the current PR and write a future kvstore-oriented PR?